src/viperleed/gui/measure/classes/downloader.py

```diff
@@ -0,0 +1,189 @@
+"""Module downloader of viperleed.gui.measure.classes.
+
+Contains NetworkHelper, a downloader that uses QtNetwork or requests as
+fallback to download data from a given url.
+"""
+
+__authors__ = (
+    'Michele Riva (@michele-riva)',
+    'Florian Dörr (@FlorianDoerr)',
+    )
+__copyright__ = 'Copyright (c) 2019-2025 ViPErLEED developers'
+__created__ = '2025-11-04'
+__license__ = 'GPLv3+'
+
+import importlib
+
+from PyQt5 import QtCore as qtc
+from PyQt5 import QtNetwork as qtn
+
+
+class _RequestsWorker(qtc.QObject):
+    """Downloder using requests module."""
+
+    # Emitted if download finished successfully.
+    finished = qtc.pyqtSignal(bytes)
+    # Emitted if download failed.
+    failed = qtc.pyqtSignal()
+
+    def __init__(self, url):
+        """Initialise downloader."""
+        super().__init__()
```

Comment:
```suggestion
    def __init__(self, url, parent=None):
        """Initialise downloader."""
        super().__init__(parent=parent)
```

Just to maintain the same interface as the `QObject` base class.